### PR TITLE
reviewwer implemnetation

### PR DESCRIPTION
**Pull request by richie1988**
### Supper Mario info App

- fetched Data from the API implemented the React roots and hooks from the react-redux
- I Used React components.
- I Used React props.
- I Used React Router
- Connect React and Redux.
- Handle events in a React app.
- Followed the font size and color and typography implementation.
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API i stored in the Redux store.
- created the map and filter to filter the data  grabbed from the API
- Every page as main page and the pages for each item.
- I implemented a backward button when the user clicks the user navigates to the home page.
- Added the link to my video live demo inside my readme.md file
- Created the link to my page as well

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo](https://richardsmarioinfo.netlify.app/)
+- [Live Demo](https://richardmarioinfo.onrender.com/home)
```
Comment (Olamarx) [inline README.md]: - [ ] After digging further, I discovered you added /home path to the url whereas, it should just be https://richardmarioinfo.onrender.com which is the root path. Kindly effect the change.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @richie1988, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linters passed ✔️
- Commit message meaningful ✔️
- Git flow ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (richie1988) [reply README.md]: Thanks

**Review by danmainah — APPROVED** [review]
Hi @richie1988 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Implemented the changes requested to fix the link error and ensure all links are working correctly.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**